variables.tf

```diff
@@ -80,3 +80,9 @@ variable "nat_gateway_enabled" {
   description = "Flag to enable/disable NAT gateways for private subnets"
   default     = "true"
 }
+
+variable "auto_assign_public_ip" {
+  type        = "string"
+  default     = "true"
+  description = "Instances launched into the subnet should be assigned a public IP address."
```

Comment:
```suggestion
  description = "Instances launched into a public subnet should be assigned a public IP address"
```

Reply:
Ah yes.. I felt that since the variable was only added to `public.tf` it wasn't justified.